docs/contributing.md

```diff
@@ -58,3 +68,15 @@ The release notes can be found [here](../release-notes) and are automatically mo
 release.
 They are also shown in the stores and
 the [release section of github](https://github.com/digitalfabrik/integreat-app/releases).
+
+### Reviews
+
+We use the following emoji code for reviewing:
```

Comment:
:clown_face: (personal opintion) I liked this in the root Readme, as I have a look at this quite often and having it directly in root makes it fast available and therefore encourages us more to use it.